test/blueprints/Do.Test.Blueprints.Service.Application/ConfigurationOverrider/ConfigurationOverriderFeature.cs

```diff
@@ -54,5 +56,10 @@ public void Configure(LayerConfigurator configurator)
         {
             applicationParts.Add(new(configurator.Context.GetGeneratedAssembly("Controllers")));
         });
+
+        configurator.ConfigureMvcNewtonsoftJsonOptions(options =>
+        {
+            options.SerializerSettings.Converters.Add(new StringEnumConverter(new CamelCaseNamingStrategy()));
```

Comment:
this was in source, you've moved it to test. move it back to source, you may use `DefaultBusinessFeature`